docs/pages/docs/guide/built-ins/cards.mdx

```diff
@@ -17,6 +19,10 @@ import { Card, Cards } from 'nextra/components'
 
 ## Usage
 
+First, import the `Card` and `Cards` components. 
+Then, import the icons that you want to use from `../../icons`.
```

Comment:
```suggestion
Then, import the icons that you want to use.
```

I think we don't need to describe `from '../../icons'` since it's just an example maybe we need to change example code to

```mdx
import { CardsIcon, OneIcon, WarningIcon } from '../path-with-your-icons'
```

@SBobbe what do you think?